src/multi_vector_simulator/server.py

```diff
@@ -97,8 +99,14 @@ def run_simulation(json_dict, **kwargs):
     evaluation.evaluate_dict(dict_values, results_main, results_meta)
 
     logging.debug("Convert results to json")
-    json_values = output_processing.store_as_json(dict_values)
-    return json_values
+
+    epa_dict_values = data_parser.convert_mvs_params_to_epa(dict_values)
+
+    output_processing.select_essential_results(epa_dict_values)
+
+    json_values = output_processing.store_as_json(epa_dict_values)
+
+    return json.loads(json_values)
```

Comment:
This means that there is only the option to store the data in EPA-format. Would it not make sense to have the option to keep the json output as it is, without dropping duplicates? This would keep all data intact and in one place for future consideration/comparision. Does ´mvs_report.py` also work with the lightweight json output?

Reply:
There are three things:
- dropping indexes of timeseries to reduce size is done anyway (store_as_json)
- convert from EPA to MVS
- drop some fields that might be redundant (project_data, energy_busses etc)

We can add an option for the server enable or not the last two. I just considered that server.py is primarily meant for MVS/EPA communication so I made this the default. 

`python mvs_tool.py -i ...` (which btw you can replace now by `mvs_tool -i ....`) calls `main` function of `cli.py`, so it is only concerned by the changes which are dropping indexes of timeseries to reduce size (store_as_json).